### ISSUE TITLE
allowing simultaneous downloads during package installation

### ISSUE THREAD

**Issue by reaseno** [microsoft/winget-cli#3246]
### Description of the new feature / enhancement

During the installation of a package using winget, the downloads of other updates in the queue are paused. This means that only after the installation is complete, the next package is loaded. However, if the downloads were allowed to proceed simultaneously without waiting for the installation to finish, it would significantly expedite the entire update process.

**Comment by mdanish-kh** [comment]
- Duplicate of https://github.com/microsoft/winget-cli/issues/225

**Comment by Trenly** [comment]
Duplicate of #225
